command/command.go

```diff
@@ -257,6 +275,20 @@ func (e *Executor) execShowTables(session *sess.Session) (exec.PullExecutor, err
 	return staticRows, errors.WithStack(err)
 }
 
+func (e *Executor) execShowSchemas(session *sess.Session) (exec.PullExecutor, error) {
+	e.metaController.DeleteEmptySchemasExcept(session.Schema)
```

Comment:
I'm not sure we should be doing this. Wouldn't it be better to delete the schema if it's empty when closing the session? Then wouldn't need this.

Reply:
So basically, as you had it previously, but with an extra call to deleteSchemaIfEmpty() in session.close() :)

Reply:
Now deleting schema when session close is called